src/index.js

```diff
@@ -73,7 +74,13 @@ export default {
 };
 
 function httpFetch(options) {
-  validateUrl(options.url);
+  if (!options.url) { // @TODO should check type
+    return Promise.reject(new Error(`Invalid url, got \`${options.url}\``));
+  }
+  if (isNotValidPath(options.url, options.params)) { // @TODO should check types
+    // @TODO should have testcase in place
```

Comment:
because of [this test](https://github.com/nordnet/nordnet-next-api/blob/master/src/__tests__/index.test.js#L47), [todo-line](https://github.com/nordnet/nordnet-next-api/blob/master/src/index.js#L88) can be removed